### PR TITLE
Share dialog

### PR DESCRIPTION
**Pull request by ShayDavidson**
screen shot 2015-03-15 at 10 32 53 am

### CODE REVIEW

File: app/src/main/java/com/paypal/psix/fragments/ShareDialogFragment.java [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+                public void onClick(DialogInterface dialog, int id) {
+                   onShare();
+                }
+            });
+        ButterKnife.inject(this, view);
+
+        urlTextField.setText(event.getShareURL());
+
+        Dialog dialog = builder.create();
+        dialog.setCanceledOnTouchOutside(false);
```
Comment (avihut) [inline app/src/main/java/com/paypal/psix/fragments/ShareDialogFragment.java:73]: Cool :)

File: app/src/main/java/com/paypal/psix/fragments/ShareDialogFragment.java [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+            public void onCompleted(Response response) {
+                finishShareFlow();
+            }
+        });
+//        requestSharePermissions(new Session.StatusCallback() {
+//            @Override
+//            public void call(Session session, SessionState sessionState, Exception e) {
+//
+//            }
+//        });
```
Comment (avihut) [inline app/src/main/java/com/paypal/psix/fragments/ShareDialogFragment.java:100]: Remove commented out code

Comment (avihut) [inline app/src/main/java/com/paypal/psix/fragments/ShareDialogFragment.java:100]: Or is this here as a basis for the permissions request solution?

File: app/src/main/java/com/paypal/psix/services/FacebookService.java [hunk 3]
```diff
@@ -35,4 +36,16 @@ public static Request getInviteesForEventRequest(String fbEventId, Request.Callb
...
+        Bundle params = new Bundle();
+        params.putString("message", message);
+        new Request(
+            Session.getActiveSession(),
+            "/" + fbEventId + "/feed",
+            params,
+            HttpMethod.POST,
+            callback
+        ).executeAsync();
+    }
```
Comment (avihut) [inline app/src/main/java/com/paypal/psix/services/FacebookService.java:50]: Awesome :+1:

File: app/src/main/java/com/paypal/psix/fragments/ShareDialogFragment.java (hunk 2 above)
Comment (ShayDavidson) [inline app/src/main/java/com/paypal/psix/fragments/ShareDialogFragment.java:100]: yes

**Comment by avihut** [comment]
Done. Looks good :ice_cream:

**Pull request merged**